cli/cmd/alpha_stat.go

```diff
@@ -176,6 +187,8 @@ func renderTrafficMetricsList(metrics *v1alpha1.TrafficMetricsList, allNamespace
 	t.Render(w)
 }
 
+// renderTrafficMetricsEdgesList will render a table to the given Writer.
+// Rows will be filtered to make sure that only traffic directed to the resource requested by the `--to` flag
```

Comment:
I think this sentence is incomplete.

Reply:
When I first read this function, I had a hard time understanding it because I didn't know what was an edge, nor its direction.

Only after reading [this](https://github.com/servicemeshinterface/smi-sdk-go/blob/master/pkg/apis/metrics/v1alpha1/edge.go) and [this](https://linkerd.io/2/reference/cli/edges/) that I had a good idea about what this function did.

Do you think we should give some kind of information about edges here? or just assume that the person that wants to contribute should already know about this?

Reply:
Off the top of my head, we can consider putting links to the edges code and documentation that you mention.

I can't think of any cases where we have links in the comments, but that doesn't mean that there aren't any :)

Reply:
I was thinking about that too... but I guess the concept of Edges will be used on other files as well. Should we provide links on every file that use edges? Do you guys think it will be too much?